google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1148,6 +1148,74 @@ def iter_crm_projects(self, parent_type, parent_id):
         for project in resources:
             yield project
 
+    # pylint: disable=inconsistent-return-statements, using-constant-test
+    def iter_crm_folder_org_policies(self, folder_id):
+        """Folder organization policy in a folder from Cloud Asset data.
+        Args:
+            folder_id (str): id of the folder to get policy.
+
+        Returns:
+            dict: Folder organization policy.
+        """
+        resource = self.dao.iter_cai_assets(
+            ContentTypes.org_policy,
+            'cloudresourcemanager.googleapis.com/Folder',
+            '//cloudresourcemanager.googleapis.com/{}'.format(folder_id),
+            self.engine)
+        if resource:
```

Comment:
the if / return seems to be used in the fetch_..() method instead of the iter_..() method. Is there any reason why this method is using this pattern? e.g. method at line 1132 and 1219.

Reply:
Updated to be consistent with other iter methods.